### PR TITLE
Remove javascript tags

### PR DESCRIPTION
**Pull request by topicus**

### CODE REVIEW

**Comment by dkinzer** [comment]
@topicus any chance you have the bandwidth to reproduce this for release-1-12 branch as well.  It would greatly improve the dkan_starter site testing time as well.

File: test/features/dataset.all.feature.orig [hunk 1]
```diff
@@ -0,0 +1,258 @@
+@api
```
Comment (dafeder) [inline test/features/dataset.all.feature.orig]: @topicus looks like you need to configure your mergetool to stop backing up: `git config --global mergetool.keepBackup false`

**Comment by topicus** [comment]
@dkinzer I'll work on that.

**Comment by dkinzer** [comment]
@topicus thanks!!!

**Pull request merged**